erpnext/stock/doctype/delivery_note/delivery_note.js

```diff
@@ -6,16 +6,48 @@
 frappe.provide("erpnext.stock");
 frappe.provide("erpnext.stock.delivery_note");
 
-frappe.ui.form.on('Delivery Note', 'onload', function(frm) {
-	frm.set_indicator_formatter('item_code',
-		function(doc) {
-			return (doc.docstatus==1 || doc.qty<=doc.actual_qty) ? "green" : "orange"
+frappe.ui.form.on("Delivery Note", {
+	onload: function(frm) {
+		var qa_no = frappe.meta.get_docfield("Delivery Note Item", "qa_no");
```

Comment:
this should be on `setup` event instead of `onload`

`onload` is called everytime a new delivery note is loaded, and `setup` is called when the form object (DOM) is created (only once)